genome/gene_statistic.py

```
@@ -0,0 +1,454 @@
+        self.pcf: Final = sf_aa_codon
+
+    @classmethod
+    def get(cls, table: int | str = 11, _table_cache={}) -> "CodonTable":
+        if table not in _table_cache:
+            _table_cache[table] = cls(table)
+        return _table_cache[table]
+
+    def parse(
+        self,
+        seq_cds: Seq,
+        transl_except: set[int] = set(),
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)